BitwardenShared/Core/Platform/Services/StateServiceTests.swift

```diff
@@ -1862,4 +1936,9 @@ class StateServiceTests: BitwardenTestCase { // swiftlint:disable:this type_body
 private struct ConnectToWatchValue: Equatable {
     let userId: String?
     let shouldConnect: Bool
+}
+
+private struct SyncToAuthenticatorValue: Equatable {
+    let userId: String?
+    let shouldSync: Bool
 } // swiftlint:disable:this file_length
```

Comment:
🤔 Is this struct really necessary? Can't we use the published tuple directly?

Reply:
It's not totally necessary but you get Equatability doing it that way which cleans up the XCTAsserts a bit:

with the struct:
``` Swift
        XCTAssertEqual(
            publishedValues,
            [
                SyncToAuthenticatorValue(userId: "1", shouldSync: false),
                SyncToAuthenticatorValue(userId: "1", shouldSync: true),
            ]
        )
```

without the struct:
``` Swift
        XCTAssertEqual(publishedValues[0].userId, "1")
        XCTAssertEqual(publishedValues[0].shouldSync, false)
        XCTAssertEqual(publishedValues[1].userId, "1")
        XCTAssertEqual(publishedValues[1].shouldSync, true)
```

The latest commit has it without the struct. See which one you prefer - I'm fine either way 👍

Reply:
Yeah, it's a pity we can't use `Equatable` with tuples, but we can use `==` operator 😄